draft-ietf-oauth-status-list.md

```diff
@@ -219,6 +220,7 @@ This section defines the structure for a CBOR-encoded Status List:
 * The `StatusList` structure is a map (Major Type 5) and defines the following entries:
   * `bits`: REQUIRED. Unsigned int (Major Type 0) that contains the number of bits per Referenced Token in the Status List. The allowed values for `bits` are 1, 2, 4 and 8.
   * `lst`: REQUIRED. Byte string (Major Type 2) that contains the Status List as specified in [](#status-list-json).
+  * `aggregation_uri`: OPTIONAL. Byte string (Major Type2) that contains an URI to retrieve the Status List Aggregation for this credential type. See section [](#batch-fetching) for further detail.
```

Comment:
Why are we repeating the definitions for CBOR Status List? In the CWT Status List Token we just reference to the JWT.Should we do the same here?

Reply:
Hmm, we need to define the major type here, but yeah we can probably make this a bit cleaner - let's briefly discuss tomorrow

Reply:
I think we should use major type 3 - text string.